docs/reference/mesh.md

```diff
@@ -29,3 +30,6 @@ If the `url` value starts with `http://` or `https://`, Webots will get the file
 Otherwise, the file should be specified with a relative path.
 The same search algorithm as for [ImageTexture](imagetexture.md) is used (cf. [this section](imagetexture.md#search-rule-of-the-texture-path)).
 Absolute paths work as well, but they are not recommended because they are not portable across systems.
+
+The `name` field defines which sub-meshes are included.
```

Comment:
From the sub-meshes (plural) it leads to believe it's possible to specify more than one, but the parameter is of type `SFString`. Is it supposed to be possible to import say 2 meshes at once in the same ColladaShape or the actual options are: all of the meshes OR just one.

Reply:
It will include all sub-meshes with the given name (I am not sure if the same names are allowed). However, I was thinking to support the * (asterisk) wildcard, I am not sure if anyone will need it.

Reply:
I believe that if the `name` string is empty (default value), it should include all the meshes instead of none to be more user friendly. Therefore, the wildcard "*" would give the same behavior as the empty string, so maybe we can drop it for now and document only the behavior with the empty string. Otherwise, people will expect that strings like "*_arm" to work, e.g., match "left_arm" and "right_arm". In such a case I am not sure which material to use... The one of the right or left arm? So, I would postpone the implementation of the wildcards until we really need them.

Reply:
I am not sure how Collada files are organized and if several meshes can use the same appearance, but it is the case, it would be more logical to merge all the meshes using the same appearance into the same IndexedFaceSet inside a Shape node. In that case the "name" parameter should refer to the appearance and not to the mesh... But I am not sure if appearance are named in Collada files...